Error "cycle detected" using default template

I tried creating a simple project with Giraffe using (in Git bash on Windows):

    $ dotnet new -i "giraffe-template::*"
    $ dotnet new giraffe
    $ ./build.sh

This fails with:

```
$ ./build.sh
  Restoring packages for C:\Temp\giraffe\src\giraffe\giraffe.fsproj...
C:\Temp\giraffe\src\giraffe\giraffe.fsproj : error NU1108: Cycle detected.
C:\Temp\giraffe\src\giraffe\giraffe.fsproj : error NU1108:   giraffe -> Giraffe (>= 3.4.0).
  Restore failed in 992.36 ms for C:\Temp\giraffe\src\giraffe\giraffe.fsproj.
Microsoft (R) Build Engine version 15.9.20+g88f5fadfbe for .NET Core
Copyright (C) Microsoft Corporation. All rights reserved.
```

My .NET version and runtims are:

```
$ dotnet --info
.NET Core SDK (reflecting any global.json):
 Version:   2.1.502
 Commit:    c74ce8f29f

Runtime Environment:
 OS Name:     Windows
 OS Version:  10.0.17763
 OS Platform: Windows
 RID:         win10-x64
 Base Path:   C:\Program Files\dotnet\sdk\2.1.502\

Host (useful for support):
  Version: 2.1.6
  Commit:  3f4f8eebd8

.NET Core SDKs installed:
  1.1.11 [C:\Program Files\dotnet\sdk]
  2.1.202 [C:\Program Files\dotnet\sdk]
  2.1.502 [C:\Program Files\dotnet\sdk]

.NET Core runtimes installed:
  Microsoft.AspNetCore.All 2.1.6 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.All]
  Microsoft.AspNetCore.App 2.1.6 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.NETCore.App 1.0.13 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 1.1.10 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 2.0.9 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 2.1.6 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
```

--- a/.gitignore
+++ b/.gitignore
@@ -294,3 +294,6 @@ __pycache__/
 #custom
 
 .temp
+
+#Ionide
+.ionide

--- a/global.json
+++ b/global.json
@@ -1,6 +1,6 @@
 {
     "projects": [ "src", "tests" ],
     "sdk": {
-      "version": "2.1.403"
+      "version": "2.2.401"
     }
 }

--- a/src/content/DotLiquid/src/AppNamePlaceholder/AppNamePlaceholder.fsproj
+++ b/src/content/DotLiquid/src/AppNamePlaceholder/AppNamePlaceholder.fsproj
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <TargetFramework>netcoreapp2.1</TargetFramework>
     <DebugType>portable</DebugType>
-    <AssemblyName>AppNamePlaceholder</AssemblyName>
+    <AssemblyName>AppNamePlaceholder.App</AssemblyName>
     <OutputType>Exe</OutputType>
     <EnableDefaultContentItems>false</EnableDefaultContentItems>
   </PropertyGroup>

--- a/src/content/DotLiquid/src/AppNamePlaceholder/AppNamePlaceholder.fsproj
+++ b/src/content/DotLiquid/src/AppNamePlaceholder/AppNamePlaceholder.fsproj
@@ -9,7 +9,7 @@
   </PropertyGroup>
 
   <ItemGroup Condition="'$(UsePaket)' == false OR '$(UsePaket)' == ''">
-    <PackageReference Include="Microsoft.AspNetCore.App" Version="2.1.*" />
+    <PackageReference Include="Microsoft.AspNetCore.App" />
     <PackageReference Include="Giraffe" Version="3.4.*" />
     <PackageReference Include="Giraffe.DotLiquid" Version="1.2.*" />
     <PackageReference Include="TaskBuilder.fs" Version="2.1.*" />

--- a/src/content/DotLiquid/tests/AppNamePlaceholder.Tests/AppNamePlaceholder.Tests.fsproj
+++ b/src/content/DotLiquid/tests/AppNamePlaceholder.Tests/AppNamePlaceholder.Tests.fsproj
@@ -5,7 +5,7 @@
 
   <ItemGroup Condition="'$(UsePaket)' == false OR '$(UsePaket)' == ''">
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.9.*" />
-    <PackageReference Include="Microsoft.AspNetCore.App" Version="2.1.*" />
+    <PackageReference Include="Microsoft.AspNetCore.App" />
     <PackageReference Include="Microsoft.AspNetCore.TestHost" Version="2.1.*" />
     <PackageReference Include="xunit" Version="2.4.*" />
     <PackageReference Include="xunit.runner.visualstudio" Version="2.4.*" />

--- a/src/content/Giraffe/src/AppNamePlaceholder/AppNamePlaceholder.fsproj
+++ b/src/content/Giraffe/src/AppNamePlaceholder/AppNamePlaceholder.fsproj
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <TargetFramework>netcoreapp2.1</TargetFramework>
     <DebugType>portable</DebugType>
-    <AssemblyName>AppNamePlaceholder</AssemblyName>
+    <AssemblyName>AppNamePlaceholder.App</AssemblyName>
     <OutputType>Exe</OutputType>
     <EnableDefaultContentItems>false</EnableDefaultContentItems>
   </PropertyGroup>

--- a/src/content/Giraffe/src/AppNamePlaceholder/AppNamePlaceholder.fsproj
+++ b/src/content/Giraffe/src/AppNamePlaceholder/AppNamePlaceholder.fsproj
@@ -9,7 +9,7 @@
   </PropertyGroup>
 
   <ItemGroup Condition="'$(UsePaket)' == false OR '$(UsePaket)' == ''">
-    <PackageReference Include="Microsoft.AspNetCore.App" Version="2.1.*" />
+    <PackageReference Include="Microsoft.AspNetCore.App" />
     <PackageReference Include="Giraffe" Version="3.4.*" />
     <PackageReference Include="TaskBuilder.fs" Version="2.1.*" />
   </ItemGroup>

--- a/src/content/Giraffe/tests/AppNamePlaceholder.Tests/AppNamePlaceholder.Tests.fsproj
+++ b/src/content/Giraffe/tests/AppNamePlaceholder.Tests/AppNamePlaceholder.Tests.fsproj
@@ -5,7 +5,7 @@
 
   <ItemGroup Condition="'$(UsePaket)' == false OR '$(UsePaket)' == ''">
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.9.*" />
-    <PackageReference Include="Microsoft.AspNetCore.App" Version="2.1.*" />
+    <PackageReference Include="Microsoft.AspNetCore.App" />
     <PackageReference Include="Microsoft.AspNetCore.TestHost" Version="2.1.*" />
     <PackageReference Include="xunit" Version="2.4.*" />
     <PackageReference Include="xunit.runner.visualstudio" Version="2.4.*" />

--- a/src/content/None/src/AppNamePlaceholder/AppNamePlaceholder.fsproj
+++ b/src/content/None/src/AppNamePlaceholder/AppNamePlaceholder.fsproj
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <TargetFramework>netcoreapp2.1</TargetFramework>
     <DebugType>portable</DebugType>
-    <AssemblyName>AppNamePlaceholder</AssemblyName>
+    <AssemblyName>AppNamePlaceholder.App</AssemblyName>
     <OutputType>Exe</OutputType>
     <EnableDefaultContentItems>false</EnableDefaultContentItems>
   </PropertyGroup>

--- a/src/content/None/src/AppNamePlaceholder/AppNamePlaceholder.fsproj
+++ b/src/content/None/src/AppNamePlaceholder/AppNamePlaceholder.fsproj
@@ -9,7 +9,7 @@
   </PropertyGroup>
 
   <ItemGroup Condition="'$(UsePaket)' == false OR '$(UsePaket)' == ''">
-    <PackageReference Include="Microsoft.AspNetCore.App" Version="2.1.*" />
+    <PackageReference Include="Microsoft.AspNetCore.App" />
     <PackageReference Include="Giraffe" Version="3.4.*" />
     <PackageReference Include="TaskBuilder.fs" Version="2.1.*" />
   </ItemGroup>

--- a/src/content/None/tests/AppNamePlaceholder.Tests/AppNamePlaceholder.Tests.fsproj
+++ b/src/content/None/tests/AppNamePlaceholder.Tests/AppNamePlaceholder.Tests.fsproj
@@ -5,7 +5,7 @@
 
   <ItemGroup Condition="'$(UsePaket)' == false OR '$(UsePaket)' == ''">
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.9.*" />
-    <PackageReference Include="Microsoft.AspNetCore.App" Version="2.1.*" />
+    <PackageReference Include="Microsoft.AspNetCore.App" />
     <PackageReference Include="Microsoft.AspNetCore.TestHost" Version="2.1.*" />
     <PackageReference Include="xunit" Version="2.4.*" />
     <PackageReference Include="xunit.runner.visualstudio" Version="2.4.*" />

--- a/src/content/Razor/src/AppNamePlaceholder/AppNamePlaceholder.fsproj
+++ b/src/content/Razor/src/AppNamePlaceholder/AppNamePlaceholder.fsproj
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <TargetFramework>netcoreapp2.1</TargetFramework>
     <DebugType>portable</DebugType>
-    <AssemblyName>AppNamePlaceholder</AssemblyName>
+    <AssemblyName>AppNamePlaceholder.App</AssemblyName>
     <OutputType>Exe</OutputType>
     <EnableDefaultContentItems>false</EnableDefaultContentItems>
   </PropertyGroup>

--- a/src/content/Razor/src/AppNamePlaceholder/AppNamePlaceholder.fsproj
+++ b/src/content/Razor/src/AppNamePlaceholder/AppNamePlaceholder.fsproj
@@ -9,7 +9,7 @@
   </PropertyGroup>
 
   <ItemGroup Condition="'$(UsePaket)' == false OR '$(UsePaket)' == ''">
-    <PackageReference Include="Microsoft.AspNetCore.App" Version="2.1.*" />
+    <PackageReference Include="Microsoft.AspNetCore.App" />
     <PackageReference Include="Giraffe" Version="3.4.*" />
     <PackageReference Include="Giraffe.Razor" Version="2.0.*" />
     <PackageReference Include="TaskBuilder.fs" Version="2.1.*" />

--- a/src/content/Razor/tests/AppNamePlaceholder.Tests/AppNamePlaceholder.Tests.fsproj
+++ b/src/content/Razor/tests/AppNamePlaceholder.Tests/AppNamePlaceholder.Tests.fsproj
@@ -6,7 +6,7 @@
 
   <ItemGroup Condition="'$(UsePaket)' == false OR '$(UsePaket)' == ''">
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.9.*" />
-    <PackageReference Include="Microsoft.AspNetCore.App" Version="2.1.*" />
+    <PackageReference Include="Microsoft.AspNetCore.App" />
     <PackageReference Include="Microsoft.AspNetCore.TestHost" Version="2.1.*" />
     <PackageReference Include="xunit" Version="2.4.*" />
     <PackageReference Include="xunit.runner.visualstudio" Version="2.4.*" />

--- a/src/giraffe-template.nuspec
+++ b/src/giraffe-template.nuspec
@@ -2,7 +2,7 @@
 <package>
     <metadata>
         <id>giraffe-template</id>
-        <version>0.20.0</version>
+        <version>0.20.1</version>
         <title>Giraffe Template for dotnet-new</title>
         <summary>A dotnet-new template for Giraffe web applications.</summary>
         <description>A dotnet-new template for Giraffe web applications.</description>

